server/routes/creatingLicences/handlers/additionalLicenceConditionUploadsHandler.ts

```diff
@@ -54,4 +63,11 @@ export default class AdditionalLicenceConditionUploadsHandler {
       `/licence/create/id/${licenceId}/additional-licence-conditions/condition/${conditionResult.id}?fromReview=true`
     )
   }
+
+  DELETE = async (req: Request, res: Response): Promise<void> => {
```

Comment:
missing test coverage for these as well

Reply:
I missed this test file completely, i'll add one now.